examples/TesterApp/webpack.config.js

```diff
@@ -1,6 +1,6 @@
 const webpack = require('webpack');
 const path = require('path');
-const ReactNative = require('../..');
+const ReactNative = require('@callstack/nativepack');
```

Comment:
This change will break tester app. There's no `@callstack/nativepack` dependency inside tester app.